src/data/AdresRegelData.ts

```diff
@@ -0,0 +1,47 @@
+import { FormFieldDefinition } from '../form/state/FormFieldDefinition.model';
+import { teletex } from './TeletexData';
+
+/**
+ * Regel 1 adres buitenland
+ *
+ * Logisch Ontwerp GBA 3.14:
+ *
+ * - elementnummer 13.30
+ * - lengte 1-35
+ */
+export const adresRegel1Validation: FormFieldDefinition = {
+  autoComplete: 'address-line1',
+  maxLength: 35,
+  minLength: 1,
+  pattern: teletex,
+};
+
+/**
+ * Regel 2 adres buitenland
+ *
+ * Logisch Ontwerp GBA 3.14:
+ *
+ * - elementnummer 13.30
+ * - lengte 1-35
+ */
+export const adresRegel2Validation: FormFieldDefinition = {
+  autoComplete: 'address-line2',
+  maxLength: 35,
+  minLength: 1,
+  pattern: teletex,
+};
+
+/**
+ * Regel 3 adres buitenland
+ *
+ * Logisch Ontwerp GBA 3.14:
+ *
+ * - elementnummer 13.30
+ * - lengte 1-35
```

Comment:
Hmm vooral voor mijn begrip: betekend lengte 1-35 ook gelijk dat hij verplicht is? Of dat als je hem invult de lengte tenminste 1 moet zijn?

Reply:
I see your point, but I wouldn't think so. The GBA doc doesn't seem to specify optional fields, even when logically they should be (huisletter, for example). So I would interpret this as: they should either be optional or adhere to this minimum length.